apps/src/music/views/MusicView.jsx

```diff
@@ -304,8 +304,9 @@ class UnconnectedMusicView extends React.Component {
     this.executeCompiledSong();
 
     Globals.setShowSoundFilters(
-      AppConfig.getValue('show-sound-filters') === 'true' ||
-        levelData?.showSoundFilters
+      AppConfig.getValue('show-sound-filters') !== 'false' &&
+        (AppConfig.getValue('show-sound-filters') === 'true' ||
+          levelData?.showSoundFilters)
```

Comment:
Why does it have the extra check for `!== 'false'`?

This looks like a boolean value, can it have something other than `true` or `false` in it?

I'm guessing this is protecting against a nullish case, where it's undefined. If so, doesn't this work:

```
AppConfig.getValue('show-sound-filters') !== 'false' || levelData?.showSoundFilters
```

It's checking to see if the value is _not_ false (so empty values default to `true`), or if the `showSoundFilters` is enabled.

Reply:
If the URL parameter is not provided, it can be `undefined`.  So this is a bit awkward, but I think both captures the ability to force the UI to be hidden, and also the ability to force it to be shown.